assets/js/components/surveys/CurrentSurvey/SurveyQuestion.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * SurveyQuestion component.
+ *
+ * Site Kit by Google, Copyright 2025 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import SurveyQuestionRating from '../SurveyQuestionRating';
+import SurveyQuestionOpenText from '../SurveyQuestionOpenText';
+import SurveyQuestionMultiSelect from '../SurveyQuestionMultiSelect';
+import SurveyQuestionSingleSelect from '../SurveyQuestionSingleSelect';
+import {
+	TYPE_MULTI_SELECT,
+	TYPE_OPEN_TEXT,
+	TYPE_RATING,
+	TYPE_SINGLE_SELECT,
+} from './constants';
+
+export default function SurveyQuestion( {
+	currentQuestion,
+	answerQuestion,
+	dismissSurvey,
+	isTheLastQuestion,
```

Comment:
How about `isLastQuestion` instead? I see this is the name of the utility so maybe that was previously a conflict. If so, I don't think it would be an issue now.

Reply:
Sounds good, updated.